boards/arm/adafruit_feather_m0_basic_proto/pinmux.c

```diff
@@ -10,10 +10,14 @@
 
 static int board_pinmux_init(const struct device *dev)
 {
-	const struct device *muxa = device_get_binding(DT_LABEL(DT_NODELABEL(pinmux_a)));
+	const struct device *muxa = DEVICE_DT_GET(DT_NODELABEL(pinmux_a));
 #if (ATMEL_SAM0_DT_SERCOM_CHECK(4, atmel_sam0_spi) && CONFIG_SPI_SAM0)
-	const struct device *muxb = device_get_binding(DT_LABEL(DT_NODELABEL(pinmux_b)));
+	const struct device *muxb = DEVICE_DT_GET(DT_NODELABEL(pinmux_b));
+
+	__ASSERT_NO_MSG(device_is_ready(muxb));
 #endif
+	__ASSERT_NO_MSG(device_is_ready(muxa));
```

Comment:
Could you move this up under the `muxa` initialization, instead of nesting the `muxb` stuff between them?  The check is unnecessarily far from the assignment.

Reply:
done.